components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -1989,6 +1998,15 @@
             "   WF_REFERENCE=?" +
             "   AND WF_TYPE=?";
 
+    public static final String GET_EXTERNAL_WORKFLOW_LIST_FOR_SUBSCRIPTIONS_SQL =
+            " SELECT " +
+                    "   WF_EXTERNAL_REFERENCE" +
+                    " FROM " +
+                    "   AM_WORKFLOWS" +
+                    " WHERE " +
+                    "   WF_REFERENCE IN (_WF_REFERENCES_)" +
```

Comment:
It is better to pass the organization id here also and let the query to internally build the _WF_REFERENCES_